services/web/server/src/simcore_service_webserver/projects/_projects_repository_legacy.py

```diff
@@ -901,7 +835,7 @@ async def get_project_product(self, project_uuid: ProjectID) -> ProductName:
                 raise ProjectNotFoundError(project_uuid=project_uuid)
             return cast(str, row[0])
 
-    async def update_project_owner_without_checking_permissions(
+    async def update_project_owner_without_checking_permissions(  # <-- Used by Garbage Collector
```

Comment:
but this could be changed to patch project no? I mean this is only changing the project owner